checker/jtreg/nullness/defaultsPersist/Fields.java

```diff
@@ -2,6 +2,7 @@
  * @test
  * @summary Test that defaulted types are stored in bytecode.
  *
+ * @requires jdk.version.major <= 24
  * @compile ../PersistUtil.java Driver.java ReferenceInfoUtil.java Fields.java
  * @run main Driver Fields
  * @ignore This fails for Java 11. See Issue 2816.
```

Comment:
@thisisalexandercook Can you look whether we can fix this? Is it a failure just on 11? Or everything >= 11?